bin/test

```diff
@@ -32,7 +32,9 @@ if [[ -n "$1" ]] && [[ $1 == "state_file" ]]; then
       spec/models/efile_security_information_spec.rb \
       spec/models/efile_submission_spec.rb \
       spec/models/efile_submission_dependent_spec.rb \
-      spec/models/efile_submission_transition_spec.rb
+      spec/models/efile_submission_transition_spec.rb \
+      spec/controllers/flows_controller_spec.rb \
+      spec/i18n_spec.rb
```

Comment:
Since the whole test suite takes too long to run on my computer, I'm still trying to get a good subset of tests that are relevant to state file. This adds the flows controller specs and the i18n_spec, both of which unexpectedly failed for me on CircleCI.